server/data/nl/pcmill.txt

```diff
@@ -0,0 +1,1195 @@
+De hacker was het computersysteem binnengedrongen.
+De spits van het team scoorde twee keer in de wedstrijd.
+Het verhullen van de fout was gebeurt om gezichtsverlies te vermijden.
+Het monster onder zijn bed bleek een kussen te zijn.
+De crimineel was in een liquidatie vermoord.
+Als je die spullen niet meer wil kan je ze beter weggeven.
+De werktafel was bedekt met bouwmateriaal.
+Door de begrenzing van zijn brommer kon hij slechts vijftig kilometer per uur rijden.
+De man weigerde meerdere keren aan het DNA-onderzoek mee te werken.
+De hijskraan was heel de dag bezig met de lading.
+Ik ben benieuwd hoe dat gaat aflopen.
+Het antwoord op de vraag verbaast me nogal.
+Kan je jezelf eerst aankleden?
+De arbeider was bezig met het afvegen van de buis.
+Het adres stond op de envelop.
+Ze was bitter geworden in de laatste jaren.
+De bewoner hield de lift bezet.
+In het café kwamen vooral stamgasten.
+De boel kan nog wel dagen branden.
+We bieden honderdduizend euro voor de kavel.
+Dat kan je wel op je buik schrijven!
+Het congres had een economisch onderwerp.
+Goedemiddag, kan ik je helpen?
+Goedenavond, waar kan ik je mee van dienst zijn?
+Voordat je de declaratie terugkrijgt moet ik het eerst goedkeuren.
+De herinnering was zo heftig dat hij er niet over wil praten.
+We hebben veel regen in juni.
+Ik heb momenteel vier ramen openstaan.
+Rockmuziek is mijn favoriete type muziek.
+Ze adviseerde hem om meteen terug te komen.
+Winkelcentra zijn geweldige plekken om te winkelen.
+Het geheugen dat we gebruikten is corrupt geraakt.
+Ik tel mijn calorieën, maar ik wil echt dessert.
+Ik kijk veel te vaak op de klok.
+Ik zal je helpen met je bagage.
+Ze was te kort om over het hek heen te kijken.
+Iedereen was druk, dus ik ging alleen naar de film.
+Ze schildert alleen met felle kleuren.
+De lucht is helder; de sterren fonkelen.
+Wij zijn nog nooit in Azië geweest.
+Hoeveel groente eet jij per dag?
+Het geweld in sommige landen kennelijk een groot probleem.
+Landbouw was een grote factor in het ontstaan van de huidige wereld.
+De leerling was het niet eens met de leraar.
+Het jacht lag in de haven, het was wel twintig meter lang.
+De kermis was een week in het dorp.
+Ik wil je bedanken voor het bedrag dat je me gaf.
+De benzine is absoluut een stuk duurder geworden.
+Als je die specifieke arts wil zien moet je dat aanvragen.
+De vader moest het scherpe object afpakken van zijn zoontje.
+De typemachine werd eigenlijk nooit meer gebruikt.
+Er was voor een aanzienlijk bedrag geïnvesteerd in de handel.
+Als je over die heuvel loopt zie je het paleis vanzelf.
+De patiënt was zo ziek dat ze haar moesten optillen.
+Persoonlijk denk ik niet dat het veel gaat opleveren.
+De subsidie is redelijk moeilijk te krijgen.
+Het slachtoffer kreeg hulp van een psychiater.
+Tijdens Kerstmis zijn er gewoonlijk minder winkels open.
+De paashaas wordt steeds populairder tijdens Pasen.
+De dromen waren heel vreemd.
+Het meisje moest huilen, ze dacht dat haar vader weg was.
```

Comment:
Zie ook wat opmerkingen verderop. Ik zou hier twee zinnen van maken maar het kan dat dit een verschil is voor BE/NL. Misschien met een ; ertussen?